lib/plugins/console/generate.js

```diff
@@ -154,13 +157,25 @@ function generateConsole(args) {
   });
 }
 
+function wrapDataStream(dataStream, options) {
+  var ignoreErrors = options && options.ignoreErrors
+
+  if (ignoreErrors) {
+    var noop = () => {}
+    dataStream.on('error', noop)
```

Comment:
This is for tests because Stream seems to detect missing lack of `error` handler. I need to check how it affects functionality.

Reply:
Ok, replaced noop with a `log.error` to keep the current behavior.